samples/snippets/src/test/java/pubsub/AdminIT.java

```diff
@@ -208,17 +249,30 @@ public void testAdmin() throws Exception {
         .contains("Created a subscription with exactly once delivery enabled:");
     assertThat(bout.toString()).contains("enable_exactly_once_delivery=true");
 
+    bout.reset();
+    // Test create a BigQuery subscription
+    createBigQueryTable();
```

Comment:
Move this to the setup step: https://github.com/googleapis/java-pubsub/blob/df8a01da0ac2f5120a745cc5013a9603cf0ccdb1/samples/snippets/src/test/java/pubsub/AdminIT.java#L92-L93